apps/web/src/interfaces/MessageInit.ts

```diff
@@ -3,4 +3,5 @@ import { ProviderString } from '@/types/ProviderString'
 export interface MessageInit {
     message: string;
     providerString: ProviderString;
+    token?: 'ETH' | 'BTC'
```

Comment:
@ccali11 Same let's find/replace all 'token' keys used in this sense to 'currency'.

Reply:
💯